docs/guides/tenant-events.mdx

```diff
@@ -19,21 +19,16 @@ of all results.
 
 ## Paging
 
-The tenant feed endpoint uses parameters that are very similar to but do not exactly match the
+The tenant feed endpoint uses parameters that match the
 [Flare standard paging pattern <Icon icon="book" size={16} />](/concepts/paging).
 
-Differences:
-- The `next` *response field* is instead named `search_after`.
-- The `from` *request parameter* is instead named `search_after`.
-
-Other than the naming difference, the function is the same:
-- Use the last response's `search_after` field as the value for the following request's `search_after` parameter.
-- Stop when the returned `search_after` is empty.
+- Use the last response's `next` field as the value for the following request's `next` parameter.
```

Comment:
Remove these lines, they no longer add any information now that we linked to paging